coordinator/src/types.rs

```diff
@@ -269,6 +269,34 @@ pub struct BlockOutcome {
 // Error code returned by check_transaction
 pub type ErrorCode = i64;
 
+/// Uniquely identifies block.
```

Comment:
Let's leave a comment here like
// FIXME: Import the `BlockId` type from the crate "foundry/types"

Reply:
I added the comment, thanks.

Reply:
Sorrry, but #365 seems to be merged soon. Let's apply the desirable change in the comment after the pr is merged. It would be merged only after the change.

Reply:
I see. I'll modify it after the pr is merged.